shifter-support-plugin.php

```diff
@@ -10,21 +10,63 @@
 License: GPL2
 */
 
+
+
+/*
+ * Shifter API
+ */
+
 require("api/shifter_api.php");
 
-define('SHIFTER_ASSET_DIR', content_url('/mu-plugins/') . basename(__DIR__) . '/assets/');
-define('SHIFTER_CSS', SHIFTER_ASSET_DIR . 'css/shifter-support.css');
-define('SWAL_JS', SHIFTER_ASSET_DIR . 'js/sweetalert.min.js');
 
-add_action("admin_enqueue_scripts", "add_shifter_support_css");
+
+/*
+ * Assets
+ * Dirs and Vars
+ */
+
+define('SHIFTER_ASSET_DIR', content_url('/mu-plugins/') . basename(__DIR__) . '/src/');
+define('SHIFTER_JS', SHIFTER_ASSET_DIR . 'js/app.js');
+define('SHIFTER_CSS', SHIFTER_ASSET_DIR . 'css/main.css');
```

Comment:
I think that you should use the file path described in `_rev-manifest.json` after building.
If you have a way to get the path like laravel-elixir, please change it.